### PR TITLE
Share dialog teams integration 8.0

### PR DESCRIPTION
**Pull request by olgarueckert-hcl**

### CODE REVIEW

File: customizer/samples/microsoft-teams/Share to Teams for CNX 7.0/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Share to Microsoft Teams
+# Share to Microsoft Teams 7.0
```
Comment (billwimer) [inline customizer/samples/microsoft-teams/Share to Teams for CNX 7.0/README.md:1]: Minor issue - could you change this to "Enabling Share to Microsoft Teams for Connections 7.0" ?

File: customizer/samples/microsoft-teams/Share to Teams for CNX 8.0/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Enabling MS Teams within Share 8.0
```
Comment (billwimer) [inline customizer/samples/microsoft-teams/Share to Teams for CNX 8.0/README.md:1]: Minor issue - could you change this to "Enabling Share to Microsoft Teams for Connections 8.0" ?

**Review by billwimer — CHANGES_REQUESTED** [review]

File: customizer/samples/microsoft-teams/Share to Teams for CNX 8.0/README.md (hunk 2 above)
Comment (olgarueckert-hcl) [reply customizer/samples/microsoft-teams/Share to Teams for CNX 8.0/README.md]: README updated

File: customizer/samples/microsoft-teams/Share to Teams for CNX 7.0/README.md (hunk 1 above)
Comment (olgarueckert-hcl) [reply customizer/samples/microsoft-teams/Share to Teams for CNX 7.0/README.md]: README updated

**Review by david-bell-hcl — APPROVED** [review]
lgtm

**Pull request merged**